ui/raidboss/popup-text.ts

```diff
@@ -380,17 +380,55 @@ class TriggerOutputProxy {
     }
 
     return value.replace(/\${\s*([^}\s]+)\s*}/g, (_fullMatch: string, key: string) => {
-      if (params !== undefined && key in params) {
-        const str = params[key];
-        switch (typeof str) {
-          case 'string':
-            return str;
-          case 'number':
-            return str.toString();
+      if (params !== undefined) {
+        let prop: string | undefined = undefined;
+
+        if (!(key in params) && key.includes('.')) {
+          const parts = key.split('.');
```

Comment:
Asking this about here and elsewhere, but is `split` correct? (Asking mostly if there should be a join on the remaining bits or an error check that there's no more `.` in the prop.) What should be do if somebody says `foo.bar.baz`? Error? Should we look for `bar` in the `foo` object (what the code does)? Should we look for `bar.baz` inside the `foo` object?